Modules/Utils/Utils/Extensions/WheelMathExtensions.swift

```diff
@@ -0,0 +1,95 @@
+import Foundation
+import CoreGraphics
+import Darwin
+
+protocol WheelMath { }
+
+extension WheelMath {
+    func angleBetween(_ point1: CGPoint, _ point2: CGPoint) -> CGFloat {
```

Comment:
I'm being nitpick now, but for 110% swiftyness 👌 
```suggestion
    func angle(Between point1: CGPoint, and point2: CGPoint) -> CGFloat {
```